packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -105,6 +113,9 @@ module.exports = (themeOptions = {}) => {
       // For querying configuration data
       'gatsby-transformer-yaml',
 
+      // For querying code samples
```

Comment:
Nit: for naming consistencies

```suggestion
      // For querying code examples
```